doc/classes/CameraServer.xml

```diff
@@ -6,7 +6,7 @@
 	<description>
 		The [CameraServer] keeps track of different cameras accessible in Godot. These are external cameras such as webcams or the cameras on your phone.
 		It is notably used to provide AR modules with a video feed from the camera.
-		[b]Note:[/b] This class is currently only implemented on macOS and iOS. On other platforms, no [CameraFeed]s will be available.
+		[b]Note:[/b] This class is currently only implemented on Linux, macOS and iOS. On other platforms, no [CameraFeed]s will be available.
```

Comment:
```suggestion
		[b]Note:[/b] This class is currently only implemented on Linux, macOS, and iOS. On other platforms, no [CameraFeed]s will be available.
```

Reply:
It should be **Linux, macOS and iOS** as now or **Linux, macOS, iOS**.
Suggested one is incorrect.

Reply:
No, please use [Oxford comma](https://docs.godotengine.org/en/latest/contributing/documentation/docs_writing_guidelines.html#use-the-oxford-comma-to-enumerate-anything).

Reply:
And this, then this should be ready

Reply:
While the Oxford comma is technically not a requirement, we already use it extensively in the Godot documentation. Therefore, we recommend new contributions to do the same for consistency 🙂

Reply:
It is a requirement as per the style page though
> Do add a comma before and or or, for the last element of a list with more than two elements.

Reply:
I meant "requirement" as per English grammar requirements.

Reply:
Done, how do I respond in future when I do not agree with suggestion? Comment is not enough?

Reply:
I mean when the comment is first made, you didn't respond on this until I reminded, I assumed you dismissed it without responding, if you forgot that's okay 🙂